src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -480,13 +483,15 @@ public StateFlow build(FlowState.Main initialState) {
                 .transitionTo(CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE).onlyIf(claimDismissedByCamunda)
             .state(CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE)
             .state(FULL_ADMISSION)
```

Comment:
https://moj.enterprise.slack.com/files/U042GKV74KZ/F04T3P5Q95X/lr_admissions_flows_v2.1.pdf?origin_team=T1L0WSW9F&origin_channel=D0477PX54GY  as per this flow this change is not required. Please confirm.

Reply:
the flow does not mention full defence. and I am not sure whether during full defence we will or not have mediation. however this story only covers part admit states paid. so i will remove this and when we cover full defence then someone will have to add it if we have mediation. At the moment with full defence we do have mediation but with different fields which is very confusing

Reply:
oh sorry Neeta, just had a look and we do have/or should have  mediation screen for full defence, so this flow applies. 
![image](https://user-images.githubusercontent.com/95431510/232703368-9f6e0f05-f1cc-45da-9d94-b8b05ff53fef.png)